engine/env/spark/pom.xml

```diff
@@ -136,6 +136,22 @@
 					<groupId>org.eobjects.metamodel-extras</groupId>
 					<artifactId>MetaModel-extras-access</artifactId>
 				</exclusion>
+                <exclusion>
```

Comment:
Hmm... Hard to imagine how to get rid of jackson here. The remote transformer in the engine-core communicates with the server via JSON-based api. And we lay on the data serialization to JSON. Any idea?